src/types.js

```diff
@@ -163,6 +171,8 @@ export type PropertyBinding = {
   internalSlot?: boolean,
 };
 
+export type LeakedObject = {| object: ObjectValue, generator: Generator |};
```

Comment:
A comment explaining the role of the generator will be very helpful to a new reader.